settings.gradle

```diff
@@ -88,3 +88,5 @@ includeWithFlags ':examples:spring-boot-minimal-kotlin',          'java'
 includeWithFlags ':examples:spring-boot-tomcat',                  'java'
 includeWithFlags ':examples:spring-boot-webflux',                 'java'
 includeWithFlags ':examples:static-files',                        'java'
+includeWithFlags ':examples:thrift-example',                      'java'
```

Comment:
Can't we just use `:examples:thrift` and `:examples:grpc` for `:examples:grpc-example`?
(for dropwizard as well)

Reply:
That sounds good!

Reply:
How about leaving a comment for grpc-example why we use the different name
with the link? https://github.com/line/armeria/pull/2236#discussion_r343020249

Reply:
It did not work because of Gradle's known bug. https://github.com/gradle/gradle/issues/847
I only renamed 'examples:thrift-example` to `example:thrift` because it is not conflict.

Reply:
Yes so how about leaving a comment so that everybody knows about it?

Reply:
Yeap, let me leave a comment :-)